src/Tools/EnvironmentUtility.php

```diff
@@ -0,0 +1,99 @@
+<?php
+declare(strict_types=1);
+
+namespace PHPSu\Tools;
+
+use PHPSu\Exceptions\CommandExecutionException;
+use PHPSu\Process\CommandExecutor;
+use PHPSu\Process\Process;
+
+final class EnvironmentUtility
+{
+    /** @var CommandExecutor  */
+    private $commandExecutor;
+
+    public function __construct(CommandExecutor $executor = null)
+    {
+        $this->commandExecutor = $executor ?? new CommandExecutor();
+    }
+
+    public function isRsyncInstalled(): bool
+    {
+        return $this->isCommandInstalled('rsync');
+    }
+
+    public function isMysqlDumpInstalled(): bool
```

Comment:
we don't need this function at this point in time, please remove it and it's tests from this PullRequest.